compiler-core/src/type_/error.rs

```diff
@@ -1297,6 +1298,7 @@ pub fn convert_get_value_constructor_error(
             location,
             name,
             variables,
+            ignored_variables: HashMap::new(),
```

Comment:
This is a lot of never-used data to pass around so let's use `discarded_location: Option<SrcSpan>` or something like that instead